Makefile

```
@@ -0,0 +1,10 @@
+
+.PHONY: test
+test : unit fuzz
+.PHONY: unit
+unit :
+	go test -v -covermode=count -coverprofile=profile.cov ./...
+.PHONY: fuzz
+fuzz :
+	go test -v -cover -fuzz=FuzzShouldIgnore ./utils -fuzztime=10s
+	go test -v -cover -fuzz=FuzzPackageCanGetVerificationCode ./utils -fuzztime=10s
```

I figured out how to disable the unit tests from running, it is to include a `-run=Fuzz` flag. We also don't need to collect coverage on the fuzzing, since there should be unit tests already that is collecting explicit test coverage. I think this is what we'd be able to use:

```suggestion
.PHONY: test
test: unit fuzz

.PHONY: unit
unit:
	go test -v -covermode=count -coverprofile=profile.cov ./...

.PHONY: fuzz
fuzz:
	go test -v -run=Fuzz -fuzz=FuzzShouldIgnore ./utils -fuzztime=10s
	go test -v -run=Fuzz -fuzz=FuzzPackageCanGetVerificationCode ./utils -fuzztime=10s
```